apps/api-documenter/src/markdown/CustomMarkdownEmitter.ts

```diff
@@ -102,40 +100,43 @@ export class CustomMarkdownEmitter extends MarkdownEmitter {
           }
         }
 
-        // write the table header (which is required by Markdown)
-        writer.write('| ');
-        for (let i: number = 0; i < columnCount; ++i) {
-          writer.write(' ');
-          if (docTable.header) {
+        writer.write('<table>');
+        if (docTable.header) {
+          writer.write('<thead><tr>');
+          for (let i: number = 0; i < columnCount; ++i) {
+            writer.write('<th>');
+            writer.writeLine();
+            writer.writeLine();
             const cell: DocTableCell | undefined = docTable.header.cells[i];
             if (cell) {
               this.writeNode(cell.content, context, false);
             }
+            writer.writeLine();
+            writer.writeLine();
+            writer.write('</th>');
           }
-          writer.write(' |');
-        }
-        writer.writeLine();
-
-        // write the divider
-        writer.write('| ');
-        for (let i: number = 0; i < columnCount; ++i) {
-          writer.write(' --- |');
+          writer.write('</tr></thead>');
         }
         writer.writeLine();
 
+        writer.write('<tbody>');
         for (const row of docTable.rows) {
-          writer.write('| ');
+          writer.write('<tr>');
           for (const cell of row.cells) {
-            writer.write(' ');
+            writer.write('<td>');
+            writer.writeLine();
+            writer.writeLine();
```

Comment:
I'm adding a few extra newlines in here, since the table cells themselves can contain markdown and I know that some markdown parsers need those to switch from "parsing HTML" back to "parsing Markdown" mode.